### PR TITLE
OpenRPAApS.OpenRPA version 1.3.43.0

### PR DESCRIPTION
**Pull request by skadefro**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40264)

### CODE REVIEW

File: manifests/o/OpenRPAApS/OpenRPA/1.3.43.0/OpenRPAApS.OpenRPA.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/open-rpa/openrpa/releases/download/1.3.39/OpenRPA.msi
```
Comment (OfficialEsco) [inline manifests/o/OpenRPAApS/OpenRPA/1.3.43.0/OpenRPAApS.OpenRPA.installer.yaml:18]: Wrong url =)

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/40264#issuecomment-1007974591

File: manifests/o/OpenRPAApS/OpenRPA/1.3.43.0/OpenRPAApS.OpenRPA.installer.yaml (hunk 1 above)
Comment (skadefro) [reply manifests/o/OpenRPAApS/OpenRPA/1.3.43.0/OpenRPAApS.OpenRPA.installer.yaml:18]: oh, thank you for catching that :-)

File: manifests/o/OpenRPAApS/OpenRPA/1.3.43.0/OpenRPAApS.OpenRPA.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/open-rpa/openrpa/releases/download/1.3.43/OpenRPA.msi
+  InstallerSha256: E4D273E3CA076859E1C972FC1DD126138C288E3AF3E850ED02147709D3927CF7
+  ProductCode: '{221C2904-9CD4-4F17-A667-37466BFF327B}'
```
Comment (OfficialEsco) [inline manifests/o/OpenRPAApS/OpenRPA/1.3.43.0/OpenRPAApS.OpenRPA.installer.yaml:20]:
```suggestion
  InstallerSha256: FD8A503B5E5BB75D18A48F6DC333A3218733D6517F18CD315A95CAD7DEC2AB5E
  ProductCode: '{FC015DE3-D378-4E78-9587-944A2C4281E8}'
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**